Path2d renderer

Fizol opened the pull request:
* In that repo is convention of additional notes that are at the start of the file, like:
```
/* ===========================================================
 * JFreeChart : a free chart library for the Java(tm) platform
 * ===========================================================
 *
 * (C) Copyright 2000-2014, by Object Refinery Limited and Contributors.
 *
 * Project Info:  http://www.jfree.org/jfreechart/index.html
 *
 * This library is free software; you can redistribute it and/or modify it
 * under the terms of the GNU Lesser General Public License as published by
 * the Free Software Foundation; either version 2.1 of the License, or
 * (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful, but
 * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
 * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
 * License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public
 * License along with this library; if not, write to the Free Software
 * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301,
 * USA.
 *
 * [Oracle and Java are registered trademarks of Oracle and/or its affiliates.
 * Other names may be trademarks of their respective owners.]
 *
 * ---------------------------
 * XYLineAndShapeRenderer.java
 * ---------------------------
 * (C) Copyright 2004-2014, by Object Refinery Limited.
 *
 * Original Author:  David Gilbert (for Object Refinery Limited);
 * Contributor(s):   -;
 *
 * Changes:
```
For me it's okey to skip it. (For examples changes statements). But I don't from the license point of view should we create similar?
* Do you want to release or working on SNAPSHOT? (for me snapshot is fine)

gdgib-roche commented:
You should add a copyright header to the files you created, much like the one shown, but with your name and Roche Sequencing Solutions in the appropriate places.  For files which are partially copied from the original code base, please include both the original author and yourself.

Fizol commented:
@gdgib-bina Thanks for reviewing it. I've pushed changes with copyright headers. I've added me as the contributor to the class I've modified. Rest of headers I've created like you said. Please review it, I want to be sure that all copyrights are appropriate.
